### ISSUE TITLE
Can not get authorization to work

### ISSUE THREAD

**Issue by elyase** [stanford-oval/genie-server#316]
I am running the Genie with:

```
docker run --name genie --rm -p 3000:3000 --env THINGENGINE_HOST_BASED_AUTHENTICATION=insecure stanfordoval/almond-server
```

After that I try a simple request:

```
curl --request POST --url 'http://127.0.0.1:3000/api/converse' --header 'content-type: application/json' --data '{
    "command": {
      "type":"command",
      "text":"what time is it?"
    }
  }'
```

But I am getting an `Unauthorized` response. What am I doing wrong?

**Comment by gcampax** [comment]
Have you completed configuration of the server? The first time the server is started (or every time if you don't use a persistent volume to store data), you need to connect with the browser and complete the configuration, regardless of the authentication setting.

**Comment by elyase** [comment]
btw is there any way to bypass the initial configuration?

**Comment by gcampax** [comment]
Yes create a prefs.db file in the directory used to store genie-server data containing the "server-login" information.
(This directory is set with the THINGENGINE_HOME env var, it defaults to /var/lib/genie-server in the standalone container)
See for example how the Home Assistant add-on does it at https://github.com/stanford-oval/almond-hassio-repository/blob/master/edge/rootfs/etc/services.d/almond/run#L11

**Comment by elyase** [comment]
nice! thanks for your help
